src/js/overview.js

```diff
@@ -100,13 +190,237 @@ $(function () {
 				.attr("height", height)
 			.append("g")
 				.attr("transform", `translate(${width/2}, ${height/2})`)
+		
+		var defs = svg.append("defs");
+		var filter = defs.append("filter")
+            .attr("id", "drop-shadow")
+            .attr("height","130%");
+
+        setFilter(filter)
+
+       function getParent(label){
+			if(label == 'serp'){
+				return 'root'
+			}
+			else{
+				return serp.dfs(label).parentId().toLowerCase()
+			}
+		}
+
+        //temporarily disables Mouse Events for a given time length 
+        function toggleMouseEvents(delay,d){
+        	mouseOut(d)
+	     	svg.selectAll("path")
+					.on("mousemove", null)
+					.on("mouseout", null)
+					.on("click",null)
+			svg.selectAll("tspan")
+				.on("mousemove", null)
+				.on("mouseout", null)
+				.on("click",null)
+			setTimeout( function(){
+				svg.selectAll("path")
+					.on("mousemove", mouseMove)
+					.on("mouseout", mouseOut)
+					.on("click",click)
+				svg.selectAll("tspan")
+					.on("mousemove", mouseMove)
+					.on("mouseout", mouseOut)
+					.on("click",click)
+			}, delay)
+        }
+
+	    function mouseMove(d) {
+	    	updateFilter(d)
+	    	setFilter(filter)
+			let text  = svg.selectAll("text").filter(text => text.name==d.name).pop()
+		 	let facet = svg.selectAll("path").filter(path => path.name==d.name).pop()
+		 	let textSize = (labelScale(d.name) * (textScale(tier)))
+		 	d3.select(text[0])
+		 		.attr('font-size', textSize + ((textSize)/10)  )
+		 		.style("text-shadow", "1px 1px 3px #fff")
+		 	d3.select(facet[0])
+	        	.style("filter", "url(#drop-shadow)");
+	        d3.select(facet[0])	
+				.transition()
+				.duration(500)
+				.ease('elastic')
+				.attr('transform',function(d){
+					var dist = 1;
+					var startAngle = getStartAngle(d)
+					var endAngle = getEndAngle(d)
+					var midAngle = ((endAngle - startAngle)/2) + startAngle;
+					var x = Math.sin(midAngle) * dist;
+					var y = Math.cos(midAngle) * dist;
+					return 'translate(' + x + ',' + y + ')';
+				});
+		}
+
+		function mouseOut(d){
+			let text  = svg.selectAll("text").filter(text => text.name==d.name).pop()
+		 	let facet = svg.selectAll("path").filter(path => path.name==d.name).pop()
+		 	d3.select(text[0])
+				.attr('font-size', d => (labelScale(d.name) * (textScale(tier))) )
+		 		.style("text-shadow", "none")
+			d3.select(facet[0])
+	      		.attr("stroke","none")
+	      		.style("filter","none");
+			d3.select(facet[0])
+				.transition()
+				.duration(500)
+				.ease('bounce')
+				.attr('transform','translate(0,0)');
+		}
+
+		function facetInfo(d){
+			var info = window.info.taxonomyInfo(d.name)
+			var explanation = document.getElementById('facet-explanation')
+			var facetTitle = document.getElementById('facet-title')
+			explanation.innerText=info[0]
```

Comment:
You should probably use an object instead of array so it's easier to follow what `info[0]` is and less prone to errors when someone updates `info` with new data

e.g. `explanation.textContent = info.description`